common/bytes.go

```diff
@@ -28,6 +29,7 @@ func ToHex(b []byte) string {
 	return "0x" + hex
 }
 
+//FromHex returns bytes of s after removing prefix, or prepends 0 if s has odd length.
```

Comment:
I think this comment is confusing more than it helps

Reply:
Hey @ligi, Apologies for my mistakes, and thank you for your feedback!